modules/cluster-wide-proxy-removal.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-cluster-wide-proxy.adoc
+
+:_content-type: PROCEDURE
+[id="cluster-wide-proxy-config_{context}"]
+= Configuring a cluster-wide proxy
+
+You can add a proxy during cluster installation. Prior to installation, however, you should verify that the proxy is accessible from the intended cluster virtual private cloud (VPC) and its private subnets.
+
+.Prerequsites
+ifdef::openshift-rosa[]
+* You have the `rosa` CLI installed and configured.
+endif::[]
+ifdef::openshift-dedicated[]
+* You have the `ocm` CLI installed and configured.
+endif::[]
+
+.Procedure
+* To create a cluster with a proxy, run the following command:
++
+ifdef::openshift-rosa[]
+[source,terminal]
+----
+$ rosa create cluster \
+ <other_arguments_here> \
+ --additional-trust-bundle-file <path_to_CA_bundle_file> \ <1> <2> <3>
+ --http-proxy http://<username>:<pswd>@<ip>:<port> \ <1> <4>
+ --https-proxy http(s)://<username>:<pswd>@<ip>:<port> <4>
+----
+endif::[]
+ifdef::openshift-dedicated[]
+[source,terminal]
+----
+$ ocm create cluster \
+ <other_arguments_here> \
+ --additional-trust-bundle-file <path_to_CA_bundle_file> \ <1> <2> <3>
+ --http-proxy http://<username>:<pswd>@<ip>:<port> \ <1> <4>
+ --https-proxy http(s)://<username>:<pswd>@<ip>:<port> <4>
+----
+endif::[]
++
+<1> The `http-proxy`, `https-proxy`, and `additional-trust-bundle-file` arguments are all optional.
+<2> If you use the `additional-trust-bundle-file` option without an `http(s)-proxy` argument, the passed additional trust bundle is set on the cluster, but it is not configured to be used with the proxy.
+<3> The `additional-trust-bundle-file` argument is a file path pointing to a bundle of PEM-encoded X.509 certificates, which are all concatenated together. The `additionalTrustBundle` field is required unless the proxy's identity certificate is signed by an authority from the {op-system} trust bundle. If you use an MITM transparent proxy network that does not require additional proxy configuration but requires additional CAs, you must provide the MITM CA certificate.
```

Comment:
@mrbarge Would you help to verify this field "additionalTrustBundle"?I'm not sure where this field is used.
Another question:when create a cluster only with --http-proxy, the CA is required,right?

Reply:
@yingzhanredhat a CA is required if the proxy is doing re-encrypting of traffic and using a CA that wouldn't be part of the system's trust store.. but that is optional.

However you raise a good point that `additionalTrustBundle file` should be renamed to `additional-trust-bundle-file parameter` 

```suggestion
<3> The `additional-trust-bundle-file` argument is a file path pointing to a bundle of PEM-encoded X.509 certificates, which are all concatenated together. The `additional-trust-bundle-file` parameter is required unless the proxy's identity certificate is signed by an authority from the {op-system} trust bundle. If you use an MITM transparent proxy network that does not require additional proxy configuration but requires additional CAs, you must provide the MITM CA certificate.
```